packages/frontend/editor-ui/src/components/canvas/Canvas.vue

```diff
@@ -992,6 +998,10 @@ provide(CanvasKey, {
 			cursor: grabbing;
 		}
 	}
+
+	&.isExperimentalNdvActive {
+		--canvas-zoom-compensation-factor: 0.67;
```

Comment:
That's a good idea ! I'm wondering if we could make this value dynamic based on scroll maybe ? (in case we change the scroll option for the embedded ndv later)

Reply:
At the moment, embedded NDV is shown at the fixed zoom level `2` (or the value set in local storage), so making this dynamic doesn't take any effect. But I think this mechanism itself could be useful outside of embedded NDV as well (e.g., showing more node description text when zoomed in). For the sake of scope, I'll keep it as an idea for the future 👍